File: src/main/java/com/sica/backend/dto/VisitRequest.java
package com.sica.backend.dto;

import jakarta.validation.constraints.NotBlank;

import java.time.LocalDateTime;

public record VisitRequest(
    @NotBlank(message = "El nombre del visitante es requerido")
    String visitorName,

    LocalDateTime visitDatetime,

    @NotBlank(message = "La persona visitada es requerida")
    String personVisited
) {}

File: src/main/kotlin/com/sica/backend/dto/VisitRequest.kt
package com.sica.backend.dto

import jakarta.validation.constraints.NotBlank
import java.time.LocalDateTime

data class VisitRequest(
    @field:NotBlank(message = "El nombre del visitante es requerido")
    val visitorName: String,
    
    val visitDatetime: LocalDateTime,
    
    @field:NotBlank(message = "La persona visitada es requerida")
    val personVisited: String
)
